src/Sylius/Component/Core/Repository/ProductRepositoryInterface.php

```diff
@@ -39,6 +39,11 @@ public function createShopListQueryBuilder(
      */
     public function findLatestByChannel(ChannelInterface $channel, string $locale, int $count): array;
 
+    /**
+     * @return array|ProductInterface[]
+     */
+    public function findByChannel(ChannelInterface $channel, int $count): array;
```

Comment:
Not needed anymore I suppose.

Reply:
Ah, sure. Thanks.

Will rebase later today

Reply:
@lchrusciel, it rebased now